lambda/custom-nft/smart-contracts/contracts/MetaKeepCollection.sol

```diff
@@ -0,0 +1,65 @@
+pragma solidity ^0.8.0;
```

Comment:
Rename this to CustomERC721 and the test file too

Reply:
Done